www/components/Docs/Variable.tsx

```diff
@@ -27,25 +31,33 @@ export default function Variable({ variable, project, anchor }: Props) {
 
 export interface VarWithTypeProps {
   name: string;
-  type?: docs.Typ;
+  type: docs.Typ | null;
```

Comment:
are you sure this is right? Does the indexer always emit `type`?

Reply:
No, but in the layer between the indexer and this we do a `.type ?? null`. I did this because currently this could be either a value, `undefined`, or `null`. The type system didn't seam to scream about `null` being passed to it, and I think it should have, so I tried to make it more explicit.